StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PollingViewController.swift

```diff
@@ -290,10 +290,7 @@ class PollingViewController: UIViewController {
         // Do one last force poll after deadline
         DispatchQueue.main.asyncAfter(deadline: .now() + 3) { [weak self] in
             guard let self = self else { return }
-            self.intentPoller.forcePoll()
-            // If we don't get a terminal status back after 20 seconds from the previous force poll, set error state to suspend polling.
-            // This could occur if network connections are unreliable
-            DispatchQueue.main.asyncAfter(deadline: .now() + 20, execute: self.setErrorStateWorkItem)
+            self.intentPoller.fetchStatus()
```

Comment:
Does this actually stop polling?  I'm not seeing how it could.

Reply:
Not really, the name of this function is misleading. Previously we depended on `viewDidDisappear` to stop polling, but honestly this function should also stop polling so I've added a call to suspendPolling in this function.